DiscordBot.DataAccess/Models/EventResponse.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace DiscordBot.DataAccess.Models
+{
+    public class EventResponse
+    {
+        public string emoji { get; }
+        public string responseName { get; }
+
+        public EventResponse(string emoji, string responseName)
+        {
+            this.emoji = emoji;
+            this.responseName = responseName;
+        }
+
+        protected bool Equals(EventResponse other)
```

Comment:
Do you need to override equals? 🙂

Reply:
I think so. Without overriding this, it will use reference equality and I think value equality is what I think will be used more regularly (considering this is basically just a named pair of two strings).

I think another option would be to convert it to a `struct` instead of a class, if you would prefer that? 🙂

Reply:
Are you comparing any event responses at the moment though? 🙂

Reply:
Not quite yet, but it will be used if anyone tries to get a specific event from the dictionary returned by `GetSignupsByResponse(...)` because this class is its key 🙂

Reply:
Okay, I would have probably just compared the fields in that case, but this is fine too. Fine to leave since it's been added, but in future I'd recommend not adding code before you need to use it 🙂